projects/data-grid/src/lib/containers/grid.component.spec.ts

```diff
@@ -1,192 +1,192 @@
-  });
-
-  it('should set the initial values', () => {
-    // given
-    const mockPagination = {
-      paginationPageSize: 0,
-      paginationPageSizeValues: [],
-      enabled: false,
-      currentPage: 0,
-      numberOfPages: 0
-    };
-
-    const expectedColumnConfig = cold('a', {a: []});
-    const expectedSelection = cold('a', {
-      a: {
-        checkboxSelection: true,
-        selectedRowsIds: []
-      }
-    });
-
-    const expectedPagination = cold('a', {
-      a: mockPagination
-    });
-
-    const expectedPagedData = cold('a', {
-      a: []
-    });
-
-    // then
-    expect(component.columnConfig$).toBeObservable(expectedColumnConfig);
-    expect(component.selection$).toBeObservable(expectedSelection);
-    expect(component.pagination$).toBeObservable(expectedPagination);
-    expect(component.pagedData$).toBeObservable(expectedPagedData);
-  });
-
-  it('should dispatch ChangePageSize when new pageSize has been chosen', () => {
-    // given
-    const pageSize = 4;
-    const action = changePageSize({pageSize});
-
-    // when
-    component.changePageSize(pageSize);
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-  it('should dispatch SortGrid action when grid sorting is called', () => {
-    // given
-    const action = sortGrid(mockConfigItem);
-
-    // when
-    component.onSortGrid(mockConfigItem);
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-  it('should dispatch ChangePageNum when a page has been selected', () => {
-    // given
-    const pageNumber = 4;
-    const action = changePageNumber({pageNumber});
-
-    // when
-    component.changePageNum(pageNumber);
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-  it('should dispatch ToggleRowSelection action when a row is toggled', () => {
-    // given
-    const index = 1;
-    const action = toggleRowSelection({rowId: index});
-
-    // when
-    component.onToggleRow(index);
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-  it('should dispatch ToggleSelectAllRows action when a all rows are toggled', () => {
-    // given
-    const action = toggleAllRowsSelection();
-
-    // when
-    component.onToggleSelectAllRows();
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-  it('should dispatch FilterGrid when filter has been updated', () => {
-    // given
-    const configWithFilter = {
-      ...mockConfigItem,
-      filter: {
-        type: FilterType.TextFilterType,
-        isFiltered: true,
-        condition: {
-          filterKey: FilteringOptions.Contains,
-          filterValue: 'aAbB'
-        }
-      }
-    };
-    const action = filterGrid(configWithFilter);
-
-    // when
-    component.changeFilterInConfig(configWithFilter);
-
-    // then
-    expect(store.dispatch).toHaveBeenCalledWith(action);
-  });
-
-});
+// TODO VV: fix test to work with the new store
```

Comment:
Is it left like this on purpose?

Reply:
Yes, it will be fixed when the grid structure will be changed